### ISSUE TITLE
Idea: incentives for volunteers

### ISSUE THREAD

**Issue by Mohammed-Shoaib** [education/graduation#2869]
Hey GitHub Education team!

I think it'd be very cool if we could receive maybe a certificate of appreciation, swag (cool T-shirt?), or anything else!

**Comment by elisemoe** [comment]
![giphy (1)](https://user-images.githubusercontent.com/6633808/82746708-b699e800-9d47-11ea-8b69-54ffb2fe00c0.gif)

**Comment by elisemoe** [comment]
We're working on it 😊

**Comment by Mohammed-Shoaib** [comment]
Thank you so much! :relaxed: You guys are truly the best. :heart:

At first I was a little depressed, but thanks to you guys, I am now super-hyped about our graduation! :tada::confetti_ball:

I sincerely thank the entire GitHub education team for making our graduation AMAZING! :fire:

I'll keep this thread open for future updates on this or any other ideas.

**Comment by puf17640** [comment]
Yeah this would be super awesome! I got a few people messaging me on instagram and facebook also asking how to contribute to the reviewing process because they saw me review quite a few

**Comment by SoumyaChoubey** [comment]
> Yeah this would be super awesome! I got a few people messaging me on instagram and facebook also asking how to contribute to the reviewing process because they saw me review quite a few

You were a diligent reviewer. I made a new PR and now after one corrected commit. No one is reviewing my application and committing it. :(

**Comment by Mohammed-Shoaib** [comment]
@SoumyaChoubey I've reviewed your application! :+1:

**Comment by Rishit-dagli** [comment]
This would be a great idea, I would love to be appreciated for my efforts in reviewing PRs :smiley:

**Comment by chandrikadeb7** [comment]
That's really great if the volunteers can have a few incentives. Thank you GitHub Education team. I have reviewed over 50 PRs I think.

**Comment by TheRaptor001** [comment]
Can i volunteer as well? Would be happy to be a part of this event :)

**Comment by TheRaptor001** [comment]
> Can i volunteer as well? Would be happy to be a part of this event :)

Realised i could still volunteer without requesting. [https://github.com/education/graduation/issues/805#issuecomment-633192151]

reviewed over 50+ PRs and glad to do more as i get time

**Comment by saarques** [comment]
You guys are getting incentives? ^_^

**Comment by Vishal1297** [comment]
> You guys are getting incentives? ^_^

:smile: :satisfied:

**Comment by Mohammed-Shoaib** [comment]
![2h42t1](https://user-images.githubusercontent.com/32389312/82830584-34383200-9ec7-11ea-92c3-505917a2143f.jpg)

**Comment by bernardbaker** [comment]
> Hey GitHub Education team!
> 
> I think it'd be cool if the volunteers could receive some form of incentive.
> 
> * Certificate of appreciation?
> * Swag, cool T-shirt?
> * Something else?
> 
> I'd love to hear your thoughts on this!

An octocat t-shirt would be amazing.

**Comment by TheRaptor001** [comment]
😻❤
